usr/sbin/pkcstok_migrate/pkcstok_migrate.c

```diff
@@ -14,13 +14,20 @@
  *
```

Comment:
FIY: On the commit message: pkcstok_migrate is not for a specific token. it can work for all tokens.
And even on AIX one can configure the tokens using the old token data format (omit `tokversion = 3.12` from opencryptoki.conf), and then later want to migrate to the new/current token data format. So this tool might still be of interest for AIX.

Reply:
Is there a real-world use case for doing this? I would assume you'd start afresh with the latest version of token data format by default if you're starting now, which is the case with all clients on AIX.

Reply:
Sure, I would also assume that users start fresh with the latest version.... but you never know what they do. Anyway, this should just work as is on AIX as well, so I don't think you need to worry about that.